scope3_methodology/cli/model_end_user_device_emissions.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+""" Compute emissions for a end user device emissions for a specific property channel """
+import argparse
+import logging
+
+from scope3_methodology.end_user_device.model import EndUserDevice
+from scope3_methodology.publisher.model import Property
+from scope3_methodology.utils.yaml_helpers import yaml_dump
+
+
+def parse_args():
+    """Parse the command line arguments"""
+    # arse command line to get company file
+    parser = argparse.ArgumentParser(description="Compute emissions for an end user device")
+    parser.add_argument(
+        "-d",
+        "--defaultsFile",
+        default="end_user_device-defaults.yaml",
+        help="Set the end user device defaults file to use",
+    )
+    parser.add_argument(
+        "-p",
+        "--propertyDefaultsFile",
+        default="property-defaults.yaml",
+        help="Set the property defaults file to use",
+    )
+    parser.add_argument("-v", "--verbose", action="store_true", help="Show derivation of output")
+    parser.add_argument("channel", nargs=1, help="The channel the device is being used within ")
+    parser.add_argument(
+        "device",
+        nargs=1,
```

Comment:
why define nargs as 1. isn't that the default?

Reply:
You actually do need the `nargs=1` without it, 
```
(scope3-methodology-py3.10) ➜ [Dec 02, 202211:22PM] ~/scope3/meth git:(add_device_emissions) ✗ python ./scope3_methodology/cli/model_end_user_device_emissions.py 'streaming' 'tablet' 
```
It interprets the first argument as `s` not `streaming`